src/composables/auth.ts

```diff
@@ -132,9 +134,20 @@ export const useAuth = createCustomScopedComposable(() => {
     }
     setLoaderVisible(false);
 
-    await openModal(MODAL_PASSWORD_LOGIN);
-    if (!encryptionKey.value) {
-      throw new Error('encryptionKey was not set after login.');
+    if (!IS_OFFSCREEN_TAB) {
+      // In development mode try to authenticate with default password
+      // If developer is not using default password, he can still login with the new password
+      if (UNFINISHED_FEATURES) {
+        // eslint-disable-next-line no-use-before-define
+        await authenticate(STUB_ACCOUNT.password).catch(async () => {
```

Comment:
The function name is `openPasswordLoginModal` but here we are trying to authenticate user before even doing this. I suggest to rename the function to something like `requestUserAuthentication` / `requestUserAuth` / `requestUserPassword` or something similar.